site/examples/plaintext.tsx

```diff
@@ -1,23 +1,29 @@
-import React, { useState, useMemo } from 'react'
+import React, { useState, useMemo, useCallback } from 'react'
 import { Node, createEditor } from 'slate'
 import { Slate, Editable, withReact } from 'slate-react'
 import { withHistory } from 'slate-history'
 
 const PlainTextExample = () => {
   const [value, setValue] = useState<Node[]>(initialValue)
   const editor = useMemo(() => withHistory(withReact(createEditor())), [])
+  const [placeholder, setPlaceholder] = useState(1)
+  const newPlaceholder = useCallback(() => setPlaceholder(p => p + 1), [])
+
   return (
-    <Slate editor={editor} value={value} onChange={value => setValue(value)}>
-      <Editable placeholder="Enter some plain text..." />
-    </Slate>
+    <>
+      <button onClick={newPlaceholder}>New Placeholder</button>
```

Comment:
This should probably be it's own example if it even makes sense. It's more of a demo of a bug fix can't be that common to change the placeholder.

Reply:
Yeah, my intention was to revert all the changes to this example before merging this PR - I just wanted a quick and easy way to demo this PR, and I don't think we'll actually want to have a changing placeholder example on master

Reply:
Oops I should have read the description more closely.

Reply:
No worries! I probably should’ve made it more prominent